src/routes/backpack/base.php

```diff
@@ -19,7 +19,28 @@
 function () {
     // if not otherwise configured, setup the auth routes
     if (config('backpack.base.setup_auth_routes')) {
-        Route::auth();
+        
+        //same routes as when calling Route::auth() but without route names
```

Comment:
Err, what about those who rely on the routes being name...and pointing at the below routes?

Reply:
Yes, you're right. 

I guess it may help to just have in a FAQ something like: 

**If you plan on having both laravel and backpack auth routes**

copy this routes to routes/backpack/base.php and remove duplicate names like 'login'
<routes here>